src/mcfret.c

```diff
@@ -0,0 +1,72 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+#include <string.h>
+#include <time.h>
+#include <fftw3.h>
+#include "omp.h"
+#include "types.h"
+#include "NISE_subs.h"
+#include "mcfret.h"
+#include "project.h"
+
+/* Distribute the MCFRET calculations */
+void mcfret(t_non *non){
+    /* Define variables and arrays */
+    /*Number of molecules*/
+    int nn2;
+    /* Response functions for emission and absorption: real and imaginary part*/
+    float *re_S_Abs,*im_S_Abs;
+    float *re_S_Emi,*im_S_Emi;
+    /*Hamiltonian of the whole system */
+    float *Hamil_i_e;
+
+
+    /*Allocate memory for all the variables*/
+    /*Allocate memory for the response functions*/
+    nn2=non->singles*(non->singles+1)/2;
+    re_S_Abs=(float *)calloc(nn2*nn2*non->tmax,sizeof(float));
+    im_S_Abs=(float *)calloc(nn2*nn2*non->tmax,sizeof(float));
+    re_S_Emi=(float *)calloc(nn2*nn2*non->tmax,sizeof(float));
+    im_S_Emi=(float *)calloc(nn2*nn2*non->tmax,sizeof(float));
+    /*Allocate memory for the Hamiltonian matrix*/
+    Hamil_i_e=(float *)calloc(nn2,sizeof(float));
+
+    /* Call the MCFRET Routine */
+    if (!strcmp(non->technique, "MCFRET") || (!strcmp(non->technique, "MCFRET-Autodetect")) || (!strcmp(non->technique, "MCFRET-Absorption"))
+        || (!strcmp(non->technique, "MCFRET-Emission")) || (!strcmp(non->technique, "MCFRET-Coupling")) || (!strcmp(non->technique,
+        "MCFRET-Rate")) || (!strcmp(non->technique, "MCFRET-Analyse")) ) {
```

Comment:
Consider changing this conform the style guide.  Line length, and one "or" on each line. Open curly bracket on new line.